src/huggingface_hub/templates/modelcard_template.md

```diff
@@ -81,15 +81,20 @@ Use the code below to get started with the model.
 
 {{ training_data | default("[More Information Needed]", true)}}
 
-## Training Procedure [optional]
+## Training Procedure 
 
 <!-- This relates heavily to the Technical Specifications. Content here should link to that section when it is relevant to the training procedure. -->
 
-### Preprocessing
+### Preprocessing [optional]
 
 {{ preprocessing | default("[More Information Needed]", true)}}
 
-### Speeds, Sizes, Times
+
+### Training hyperparameters
+- **Precision:** [More Information Needed] <!--fp16, bf16, fp8, fp32 -->
```

Comment:
Should we surface these up as Jinja Template kwargs so folks can programmatically add?

Reply:
:+1: on this. Would be something like:

```suggestion
- **Training regime:** {{ training_regime | default("[More Information Needed]", true)}} <!--fp32, fp16 mixed precision, bf16 mixed precision, bf16 non-mixed precision, fp16 non-mixed precision, fp8 mixed precision -->
```